automation.py

```diff
@@ -0,0 +1,17 @@
+import unittest
+from flask_init import app
+class Tester(unittest.TestCase):
+	def test_init_server(self):
+		tester = app.test_client(self)
+		response = tester.get('/', content_type='html/text')
+		self.assertEqual(response.status_code, 200)
+		
+	def test_Failure(self):
+		tester = app.test_client(self)
+		response = tester.get('a', content_type='html/text')
+		self.assertEqual(response.status_code, 404)
```

Comment:
a 404 error is a specific check that a page doesn't exist, but it assumes the rest of the application is working. If the server isn't on you won't get any status_code, it will be a connection refused.